packages/playwright-core/src/server/browserContext.ts

```diff
@@ -276,6 +276,22 @@ export abstract class BrowserContext extends SdkObject {
     return await this.doGetCookies(urls as string[]);
   }
 
+  async removeCookies(cookies: {name?: string, domain?: string, path?: string}): Promise<void> {
+    const setCookies = await this.cookies();
+
+    if (!cookies.name && !cookies.domain && !cookies.path)
+      return;
+
+    const filteredCookies = setCookies.filter(cookie => {
```

Comment:
maybe naming this `cookiesToKeep` ?

Reply:
@Smrtnyk: We discourage stylistic review comments, especially from non-contributors.

Reply:
Got it! just pushed outdated fixes :(
let me quickly get things straightened out